lib/matplotlib/cbook.py

```diff
@@ -715,6 +715,17 @@ def is_sequence_of_strings(obj):
     return True
 
 
+def is_hashable(obj):
```

Comment:
That only works for built-in types or types that explicitly use the `collections.Hashable` abc.  This implementation is more "duck-typing" style and therefore includes more things.